bulk-export-client/utils.bal

```diff
@@ -19,6 +19,25 @@ import ballerina/task;
 import ballerinax/health.fhir.r4;
 import ballerinax/health.fhir.r4.international401;
 
+
+# Create HTTP client with optional authentication.
+#
+# + serverConfig - Server configuration containing auth details
+# + return - HTTP client instance or error
+public isolated function createHttpClient(BulkExportServerConfig serverConfig) returns http:Client|error {
+    if serverConfig.authEnabled {
+        http:OAuth2ClientCredentialsGrantConfig config = {
+            tokenUrl: serverConfig.tokenUrl ?: "",
```

Comment:
should validate tokenUrl is empty? and client id secret as only the client credentials are supported?

Reply:
no need to ref impl